sos/collector/clusters/pacemaker.py

```diff
@@ -11,7 +11,10 @@
 import re
 
 from sos.collector.clusters import Cluster
-from setuptools._vendor.packaging import version
+try:
+    from setuptools._vendor.packaging import version
+except SyntaxError:
+    from packaging import version
```

Comment:
Nice find.

Looking at this, we may want to drop the `from setuptools._vendor` import, and instead mirror the use of `parse_version` like we're (now) doing in `sosnode`. This would give us some consistency at least.

The `pacemaker` check here is a simple one-off check to handle how an option is formatted. So changing to something like:

```diff
@@ -11,7 +11,7 @@
 import re
 
 from sos.collector.clusters import Cluster
-from setuptools._vendor.packaging import version
+from pkg_resources import parse_version
 from xml.etree import ElementTree
 
 
@@ -63,7 +63,7 @@ class pacemaker(Cluster):
         _ver = self.exec_primary_cmd('crm_mon --version')
         if _ver['status'] == 0:
             cver = _ver['output'].split()[1].split('-')[0]
-            if not version.parse(cver) > version.parse('2.0.3'):
+            if not parse_version(cver) > parse_version('2.0.3'):
                 xmlopt = '--as-xml'
         else:
             return
```

Would be an easy replacement.

Alternatively, if we're going to be doing this in multiple spots, we could simply add a wrapper in `utilities.py`.

Reply:
ack, will make it consistent, just waiting for all the tests to pass, and will push an update